samples/Java/Basic/SimpleSend/src/main/java/com/microsoft/azure/eventhubs/samples/SimpleSend/SimpleSend.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project root for full license information.
+ */
+package com.microsoft.azure.eventhubs.samples.SimpleSend;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.microsoft.azure.eventhubs.ConnectionStringBuilder;
+import com.microsoft.azure.eventhubs.EventData;
+import com.microsoft.azure.eventhubs.EventHubClient;
+import com.microsoft.azure.eventhubs.PartitionSender;
+import com.microsoft.azure.eventhubs.EventHubException;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.time.Instant;
+import java.util.Random;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ExecutorService;
+
+public class SimpleSend {
+
+    public static void main(String[] args)
+            throws EventHubException, ExecutionException, InterruptedException, IOException {
+
+        final ConnectionStringBuilder connStr = new ConnectionStringBuilder()
+                .setNamespaceName("Your Event Hubs namaspace name") // to target National clouds - use .setEndpoint(URI)
+                .setEventHubName("Your event hub")
+                .setSasKeyName("Your policy name")
+                .setSasKey("Your primary SAS key");
+
+        final Gson gson = new GsonBuilder().create();
+
+        final ExecutorService executorService = Executors.newSingleThreadExecutor();
```

Comment:
add a comment here to provide guidance on executorservice.
mention that - we used single thread executor - as there is only 1 EHclient created for this simple sample. But, in general - customers will need to think thru and model their thread pool size - and can share the ExecutorService across multiple EHClient instances.

Reply:
this could probably cater to all samples.

Reply:
will try to keep this sample simple and add the details to the other send options.

Reply:
I expect this sample to be used the most! so, please see if you can add this critical detail.

Reply:
Added. Let me if it sounds about right. And for PS and CLI scripts for Geo-DR, they have been tested before adding here :)